plugins/commands/downloader/src/main/kotlin/DownloaderCommand.kt

```diff
@@ -147,7 +147,7 @@ class DownloaderCommand(descriptor: PluginDescriptor = DownloaderCommandFactory.
     private val vcsPath by option(
         "--vcs-path",
         help = "The VCS path if '--project-url' points to a VCS. Ignored if '--ort-file' is also specified. " +
-            "(default: the empty root path)"
+            "(default: the root path)"
```

Comment:
I think the description is still missing the point to explain what this is used for, compare with:
https://github.com/oss-review-toolkit/ort/blob/d8ebd622b2aa2bd533aba744b52d16f0ed460477/model/src/main/kotlin/VcsInfo.kt#L43-L46

So if you are on it anyway, maybe rewrite it completely?

Reply:
Hmm, that description certainly is too long for a CLI help. But I could say "The VCS path to limit the checkout to ...", if that helps?

Reply:
Yes, anything that does not require the user to know what a "VCS path" is used for.